tests/integration-tests/tests/cli_commands/test_cli_commands.py

```diff
@@ -74,10 +76,18 @@ def _test_pcluster_stop_and_start(cluster, region, expected_num_nodes):
     # Sample pcluster start output:
     # Compute fleet status is: STOPPED. Submitting status change request.
     # Request submitted successfully. It might take a while for the transition to complete.
-    # Please run 'pcluster status' if you need to check compute fleet statu
+    # Please run 'pcluster status' if you need to check compute fleet status
     expected_start_output = (
         r"Compute fleet status is: STOP.*Submitting status change request.*" "\nRequest submitted successfully"
     )
     assert_that(cluster_start_output).matches(expected_start_output)
     wait_for_num_instances_in_cluster(cluster.cfn_name, region, desired=expected_num_nodes)
     _test_pcluster_instances_and_status(cluster, region, compute_fleet_status="RUNNING")
+
+
+def _test_pcluster_export_cluster_logs(s3_bucket_factory, cluster):
+    """Test pcluster export-cluster-logs functionality."""
+    logging.info("Testing that pcluster export-cluster-logs is working as expected")
+    bucket_name = s3_bucket_factory()
+    with tempfile.TemporaryDirectory() as tempdir:
+        cluster.export_logs(bucket_name, tempdir)
```

Comment:
is there any further validation that can be performed on the output of the command? maybe check that an archive exists locally somewhere, or that objects were saved (or deleted) in S3 as expected?

Reply:
Added some checks for archive existence and content and s3 cleanup. Thanks, good point.